analyzers/tests/SonarAnalyzer.Test/TestCases/TooManyLabelsInSwitch.cs

```diff
@@ -56,9 +143,108 @@ public TooManyLabelsInSwitch(int n, MyEnum en)
             }
         }
 
+        public int SwitchCase(char ch, int value)
+        {
+            switch(ch)  // Noncompliant {{Consider reworking this 'switch' to reduce the number of 'case' clause to at most 2 or have only one statement per 'case'.}}
+//          ^^^^^^
+            {
+                case 'a':
+                    return 1;
+                case 'b':
+                    return 2;
+                case 'c':
+                    throw new NotImplementedException();
+                // ...
+                case '-':
+                    if (value > 10)
+                    {
+                        return 42;
+                    }
+                    else if (value < 5 && value > 1)
+                    {
+                        return 21;
+                    }
+                    return 99;
+                default:
+                    return 1000;
+            }
+        }
+
+        public int SwitchCaseWithCodeBlock(char ch, int value)
+        {
+            switch(ch)  // Noncompliant {{Consider reworking this 'switch' to reduce the number of 'case' clause to at most 2 or have only one statement per 'case'.}}
+//          ^^^^^^
+            {
+                case 'a':
+                    return 1;
+                case 'b':
+                    throw new NotImplementedException();
+                case 'c':
+                    return 3;
+                // ...
+                case '-':
+                    {
+                        if (value > 10)
+                        {
+                            return 42;
+                        }
+                        else if (value < 5 && value > 1)
+                        {
+                            return 21;
+                        }
```

Comment:
This is distracting. Replace it with two simple statements. Same in the test below.

Reply:
I would like to keep this one as it helped me identify that `if ... else if` is a single statement.

Reply:
But wouldn't the one above do that already?

Reply:
Right, I forgot I duplicated this section, I will make the update